rust/src/core.rs

```diff
@@ -143,6 +144,36 @@ pub(crate) fn refspec_classify(refspec: &str) -> crate::ffi::RefspecType {
     }
 }
 
+/// Run systemd-sysusers in the rootfs if the %__systemd_sysusers RPM macro is set.
+pub(crate) fn run_sysusers(rootfs_dfd: i32) -> CxxResult<()> {
+    let args: Vec<_> = vec!["rpm", "--eval=%{?__systemd_sysusers}"]
+        .into_iter()
+        .map(|s| s.to_string())
+        .collect();
+    let path = crate::bwrap::bubblewrap_run_sync(
+        rootfs_dfd,
+        &args,
+        true,
+        crate::ffi::BubblewrapMutability::Immutable,
+    )?;
+    let tmp = String::from_utf8(path).expect("Could not read %__systemd_sysusers as UTF8");
+    let cmd = tmp.trim_end_matches(&['\r', '\n'][..]);
+    if !cmd.is_empty() {
+        progress_task("Running systemd-sysusers", || -> CxxResult<()> {
+            let args: Vec<_> = vec![cmd].into_iter().map(|s| s.to_string()).collect();
```

Comment:
```suggestion
            let args: Vec<_> = vec![cmd.into()];
```

At that point, I'd probably just inline it at the parameter below.